package.json

```diff
@@ -141,46 +141,46 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.10.4",
+    "@babel/core": "^7.11.1",
     "@babel/plugin-proposal-object-rest-spread": "^7.10.1",
     "@babel/plugin-proposal-throw-expressions": "^7.10.1",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/plugin-transform-runtime": "^7.10.4",
-    "@babel/polyfill": "^7.10.1",
-    "@babel/preset-env": "^7.10.2",
+    "@babel/plugin-transform-runtime": "^7.11.0",
+    "@babel/polyfill": "^7.10.4",
+    "@babel/preset-env": "^7.11.0",
     "@babel/preset-react": "^7.10.1",
-    "@babel/runtime": "^7.10.4",
+    "@babel/runtime": "^7.11.1",
     "@bbc/apache2-license-checker": "^1.1.4",
     "@bbc/psammead-storybook-helpers": "^8.3.4",
     "@bbc/psammead-test-helpers": "^4.0.0",
     "@loadable/babel-plugin": "^5.12.0",
     "@storybook/addon-knobs": "^5.3.19",
     "@storybook/react": "^5.3.19",
     "@storybook/theming": "^5.3.19",
-    "@testing-library/dom": "^7.16.2",
-    "@testing-library/jest-dom": "^5.10.1",
-    "@testing-library/react": "^10.3.0",
+    "@testing-library/dom": "^7.21.8",
+    "@testing-library/jest-dom": "^5.11.2",
+    "@testing-library/react": "^10.4.8",
     "@testing-library/react-hooks": "^3.3.0",
     "amphtml-validator": "^1.0.31",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^26.0.1",
     "babel-loader": "^8.1.0",
     "babel-plugin-dynamic-import-node": "^2.3.3",
-    "babel-plugin-styled-components": "^1.10.7",
+    "babel-plugin-styled-components": "^1.11.1",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "bbc-a11y": "^2.4.2",
     "brotli-webpack-plugin": "^1.1.0",
     "chalk": "^4.1.0",
-    "chrome-launcher": "^0.13.3",
+    "chrome-launcher": "^0.13.4",
     "compression-webpack-plugin": "^4.0.0",
-    "copy-webpack-plugin": "^6.0.2",
+    "copy-webpack-plugin": "^6.0.3",
     "cypress": "^4.10.0",
     "cypress-multi-reporters": "^1.4.0",
     "depcheck": "^1.0.0",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "enzyme-to-json": "^3.5.0",
-    "eslint": "^7.2.0",
+    "eslint": "^7.6.0",
```

Comment:
This isn't a major revision, e.g. 7.x.x -> 8.x.x, but since it's several .x versions behind, are there any special considerations to make here and would it be worth pulling into its own PR?

Reply:
Tests ran and all linting pasted. So I'd say its good to go 👍